features/environment.py

```diff
@@ -3,5 +3,13 @@
 from behave.model import Scenario
 
 def before_feature(context, feature):
+    # @tfk we have this strange issue between stack frames blending over
+    # between features so we need to preserve only the bottom two stack
+    # frames when beginning a new feature.
+    context._stack = context._stack[-2:]
```

Comment:
Aah, that explains it .. I tried to add context._push() with at the 'Its values' statement to retrieve it later and run into exactly this (another feature somehow not working anymore ..).